.babelrc

```
@@ -3,6 +3,12 @@
 	"@wordpress/babel-preset-default"
   ],
   "plugins": [
-	"@babel/plugin-proposal-class-properties"
+	"@babel/plugin-proposal-class-properties",
+  [
+    "@babel/plugin-transform-runtime",
+    {
+      "corejs": 2,
+    }
+  ]
```

```suggestion
 	"@babel/plugin-proposal-class-properties"
```
With the updates in 5.8-beta3 these are no longer necessary.